kvbc/src/reconfiguration_kvbc_handler.cpp

```diff
@@ -567,8 +567,11 @@ bool ReconfigurationHandler::handle(const concord::messages::ClientKeyExchangeSt
   for (const auto& gr : bftEngine::ReplicaConfig::instance().clientGroups) {
     for (auto cid : gr.second) {
       if (command.tls) {
-        std::string client_cert_path = bftEngine::ReplicaConfig::instance().certificatesRootPath + "/" +
-                                       std::to_string(cid) + "/client/client.cert";
+        std::string client_cert_path =
+            (bftEngine::ReplicaConfig::instance().useUnifiedCertificates)
```

Comment:
Nit: define const std::string base_path = bftEngine::ReplicaConfig::instance().certificatesRootPath;
to make an operator below shorter.